openapi/components/schemas/RiskMetadata.yaml

```
@@ -114,8 +114,8 @@ properties:
     description: Specifies if the customer's IP address is related to hosting.
     type: boolean
     readOnly: true
-  vpnServiceName:
-    description: VPN service name, if available.
+  dch:
```

```suggestion
  hostingName:
```
isHosting is related to this field, so I wonder if we should have consistent naming?